README.md

```diff
@@ -1,5 +1,7 @@
 # demo-springboot-database
 
+[![Coverage Status](https://coveralls.io/repos/github/florentin-a-p/demo-springboot-database/badge.svg)](https://coveralls.io/github/florentin-a-p/demo-springboot-database)
```

Comment:
Add coverage badge, so that you (and everyone) know the current state of the coverage.

To use Coveralls, you can register, then add the repo into Coveralls, then add the token into repo's secrets.